x/attribute/types/keys.go

```diff
@@ -27,13 +29,35 @@ var (
 	// Legacy amino encoded objects use this key prefix
 	AttributeKeyPrefixAmino      = []byte{0x00}
 	AttributeKeyPrefix           = []byte{0x02}
-	AttributeKeyPrefixAddrLookup = []byte{0x03}
+	AttributeAddrLookupKeyPrefix = []byte{0x03}
+	AttributeExpirationKeyPrefix = []byte{0x04}
 )
 
 // AddrAttributeKey creates a key for an account attribute
 func AddrAttributeKey(addr []byte, attr Attribute) []byte {
 	key := AttributeKeyPrefix
-	key = append(key, address.MustLengthPrefix(addr)...)
+	return append(key, AddrAttributeKeySuffix(addr, attr)...)
+}
+
+// AddrAttributeKeySuffix returns attribute key without AttributeKeyPrefix
+func AddrAttributeKeySuffix(addr []byte, attr Attribute) []byte {
```

Comment:
A key suffix usually isn't worthwhile, but the prefix is handy for iteration. Is there a reason to go this `Suffix` route instead of the standard `Prefix` way?

Reply:
Ah, interesting.  I removed the part that actually needed to use that method.  Good catch I can remove that.